emails/scoreboard-welcome.heml

```diff
@@ -0,0 +1,43 @@
+<heml>
+  <head>
+    <style>
+      body {
+        background: #1e5a8d;
+        padding: 50px 5px;
+      }
+      block {
+        background: #ffffff;
+        padding: 25px;
+      }
+      p {
+        font-family: "lucida sans unicode", "lucida grande", sans-serif
+      }
+    </style>
+  </head>
+  <body>
+    <container>
+      <block>
+        <img src="" />
+      </block>
+      <block>
+        <p>Hello ,</p>
+        <p>Congratulations! 🎉 You have earned your first SSE membership for this semester.</p>
```

Comment:
it doesn't appear that your :tada: emoji got persisted

Reply:
[Huh?](https://github.com/rit-sse/node-api/pull/74/files#diff-701d4bea5aff9022464772fc5ee49952R109)